source/systems/common.rst

```diff
@@ -64,5 +64,1789 @@ RDHPCS Platforms
+
+To request multiple partitions, list the partitions separated by a comma. For
+example, on Jet to use only the xJet and kJet general compute resource
+partitions, specify:
+
+.. code-block:: shell
+
+   #SBATCH --partition=xjet,kjet
+
+Specifying the number of cores for your job
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use `--ntasks`` to set the total number of cores required for your job.
+
+.. code-block:: shell
+
+   #SBATCH --ntasks=12
+
+This is the correct way to set the number of procs (cores) needed to run a job.
+If you need to change the processor layout, refer to the next section for
+instructions. For the maximum core count allowed per QOS, see the table below.
+
+Specifying a processor layout for your job (uniform layout example)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The simple method of laying out tasks where all the cores on a node are used
+with one MPI task per core works reasonably well for most applications.  These
+are however cases where the default amount of memory available per core is
+insufficient and need more memory than is available.
+
+In those instances, it will be necessary to spread out tasks on more nodes so
+that there are fewer MPI tasks on a node than there are cores.  The other cores
+may be left idle or could be used for speeding up the code by using threads.
+
+For example on a machine that has 12 cores per node, the default layout
+mentioned above would use all the 12 cores per node.  If each task needs twice
+the amount of memory, then you would have to place only 6 MPI tasks on each
+node.
+
+In the example below, even though there are 12 cores available on the node,
+only 6 MPI tasks are placed on a node so that each task gets double the amount
+of memory than using all the 12 cores.
+
+.. code-block:: shell
+
+   #SBATCH --nodes=4
+   #SBATCH --ntasks-per-node=6
+
+The --cpus-per-task option can be used to specify layout for a threaded job
+(e.g. OpenMP). For example, a hybrid MPI/OpenMP job where each MPI process uses
+2 threads:
+
+.. code-block:: shell
+
+   #SBATCH --nodes=4
+   #SBATCH --ntasks-per-node=3
+   #SBATCH --cpus-per-task=2
+
+
+   export OMP_NUM_THREADS=2          # Note that this is needed too!
+   srun ./myexe
+
+Other examples:
+
+.. code-block:: shell
+
+   #SBATCH --nodes=12
+   #SBATCH --ntasks-per-node=1
+
+The above example will start the job on 12 nodes, with one task/thread per
+node.
+
+.. note::
+
+   **--nodes=20** is not the same as **-nodes=20 --ntasks-per-node=12**. By
+   default, 1 task per node is used. It is best to always explicitly list the
+   --ntasks-per-node (or --ntasks) expression that you need.
+
+
+**It is required that the user specify the number of tasks they want, either
+with -n (--ntasks) or -N (--nodes) or both.** Failing to specify the number of
+tasks will result in a job submission error.
+
+Example for WRF
+"""""""""""""""
+
+WRF is a good example where non-uniform placement is required, as the first
```

Comment:
I cannot follow this.  This needs to be rewritten.

Reply:
removed per Raghu